src/theory/bv/theory_bv.cpp

```diff
@@ -200,7 +200,16 @@ Theory::PPAssertStatus TheoryBV::ppAssert(
   return d_internal->ppAssert(tin, outSubstitutions);
 }
 
-TrustNode TheoryBV::ppRewrite(TNode t) { return d_internal->ppRewrite(t); }
+TrustNode TheoryBV::ppRewrite(TNode t)
```

Comment:
Does this change the default behavior for the BV theory? That is, when will `texp` be null?

Reply:
It impacts when expand definitions is applied during preprocessing.  It was previously the case that expandDefinitions (for BV) is applied at the beginning of preprocessing during expandDefinitions, now it is applied at the end (during theory-preprocess / ppRewrite).

Reply:
So this only affects preprocessing passes then, right?

Reply:
It impacts preprocessing and `get-value`/`get-model`.